examples/django/slackapp/models.py

```diff
@@ -249,12 +254,34 @@ def consume(self, state: str) -> bool:
 )
 
 
-@app.event("app_mention")
-def event_test(body, say, logger):
+def event_test(body, say, context: BoltContext, logger):
     logger.info(body)
-    say("What's up?")
+    say(":wave: What's up?")
+
+    found_rows = list(
```

Comment:
If there is no thread-bound database connection when first access to any Django models, Django automatically establishes a new connection and associates it with the thread. This example demonstrates the situation where a listener performs a database query using Django ORM in a different thread (=thread managed by Bolt, not Django).